test/jdk/java/awt/Checkbox/CheckboxCheckerScalingTest.java

```diff
@@ -72,7 +74,8 @@ public static void main(String[] args) throws Exception {
                 check: {
                     for (int i = 0; i < imageAfterChecked.getHeight(); i++) {
                         for (int j = 0; j < imageAfterChecked.getWidth(); j++) {
-                            if (Color.black.getRGB() == imageAfterChecked.getRGB(i, j)) {
+                            Color pixelColor = new Color(imageAfterChecked.getRGB(i, j));
+                            if (compareColor(pixelColor)) {
                                 checkmarkFound = true;
                                 break check;
```

Comment:
@TejeshR13 Logging the pixelColor inside the if block can be useful for debugging purpose.
You could adda counter variable here and continue until 4-5 pixels match the criteria.

Reply:
On failure we are saving the image, I don't think logging pixelColor would add any additional help in debugging here.

Reply:
> On failure we are saving the image, I don't think logging pixelColor would add any additional help in debugging here.

True, this is not for the failure case but to get some idea on which pixel colors are picked up with the current color tolerance and to adjust the tolerance value accordingly after running CI tests. It is not required though.